client/account-settings/index.js

```diff
@@ -2,12 +2,12 @@
  * External dependencies
  */
 import React from 'react';
-import { combineReducers } from 'redux';
+import { combineReducers } from 'state/utils';
```

Comment:
Yes, it's imported from Calypso (I forgot to move it to `Internal dependencies`). If you call it without an `schema` argument, it's basically equivalent to the `redux` method, so there's no reason to *not* use it.
Moved those imports to `Internal imports` in https://github.com/Automattic/woocommerce-services/pull/1051/commits/fa305cab23ed07de90ba0b01eeeb90428f1b18d0